lib/Runtime/Library/JavascriptLibrary.cpp

```diff
@@ -3890,30 +3893,26 @@ namespace Js
 
         if (scriptConfig->IsES6RegExPrototypePropertiesEnabled())
         {
-            RuntimeFunction* globalGetter = library->CreateGetterFunction(PropertyIds::global, &JavascriptRegExp::EntryInfo::GetterGlobal);
-            library->AddAccessorsToLibraryObject(regexPrototype, PropertyIds::global, globalGetter, nullptr);
-            library->regexGlobalGetterFunction = globalGetter;
-
+            library->regexGlobalGetterFunction =
```

Comment:
You're right, the initial slot capacity is wrong here. I changed it to `16` though, including the properties behind the `ES6RegExPrototypeProperties` flag.